test/workbox-background-sync/sw/lib/test-QueueDb.mjs

```diff
@@ -350,6 +350,26 @@ describe(`QueueDb`, () => {
     });
   });
 
+  describe('getEntryCountByQueueName', () => {
+    it(`should return the number of entries in IDB filtered by index`, async () => {
+      const queueDb = new QueueDb();
+
+      await queueDb.addEntry(entry1);
+      await queueDb.addEntry(entry2);
+      await queueDb.addEntry(entry3);
+      await queueDb.addEntry(entry4);
+      await queueDb.addEntry(entry5);
+
+      expect(await queueDb.getAllEntriesByQueueName('a')).to.equal(3);
```

Comment:
I think you mean to call `getEntryCountByQueueName()` here and later on on lines 368 and 369, but instead you're  calling `getAllEntriesByQueueName()`.

Reply:
You're right, must've missed this as I wrote this on Windows and (judging the CI) it only runs a subset of the tests.
Will fix this.